libcontainer/seccomp/seccomp_linux.go

```diff
@@ -266,3 +266,8 @@ func parseStatusFile(path string) (map[string]string, error) {
 
 	return status, nil
 }
+
+// GetLibseccompVersion returns major, minor, and micro.
+func GetLibseccompVersion() (uint, uint, uint) {
```

Comment:
since 
 - the package name is already `seccomp`;
 - golang style discourages the `Get` prefix;

maybe just `Version` (or `LibVersion`, or `LibraryVersion`).

Reply:
> golang style discourages the `Get` prefix

This is what I mean: https://golang.org/doc/effective_go.html#Getters

Reply:
updated